### DISCUSSION TITLE
Mod select sounds stack uncomfortably loud

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#22264] [Ideas] [answered]
turn on sound to 3/4 if youre sensitive

https://user-images.githubusercontent.com/87947227/212988146-fce78fb2-e01d-492a-a28a-418c340bebfb.mp4

as you can hear the more mods are selected within a preset, thel louder it gets since all sounds are porbably played simultaniously, maybe have these presets just make their own sound and mute the ones in the background.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/20091

**Reply by Inconsist** [reply]
I just want to state that to me it looks like I opened most of the q&a's in here in the past days, and I really try to find if they're already known, but i didnt find the existing issue this time. If you say its to much or always to small details I'll do less. I tried to mitigate that by packing many questions in one post so its a lower frequency for you, but peppy told me to make one for each and here we are. I dont want to overload you with issues that "slow down" development rather than aid it. I just play alot and notice details like doubling scores or actually most recently my play showing up twice in the leaderboard in the horizontal scroll view right after the play was set... so, if you want me to slow down and not report every detail I will.

**Reply by bdach** [reply]
Don't worry about it. Core team has collectively turned off notifications so we don't get spammed, and we browse issues and discussions at our own pace.

Better to post something and have it turn out to be a duplicate than not post something that might not be known.
